tools/typespec-bump-deps/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@azure-tools/typespec-bump-deps",
+  "version": "0.9.6",
+  "description": "Tool to grab latest @next version for all Typespec packages",
+  "main": "dist/index.js",
+  "type": "module",
+  "bin": {
+    "typespec-bump-deps": "./cmd/cli.mjs"
+  },
+  "scripts": {
+    "watch": "tsc -p ./tsconfig.build.json --watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "clean": "rimraf dist/ temp/",
+    "test": "vitest run"
```

Comment:
do we run test in ci? i have no special requirement for config and package files since they are just template codes.

Reply:
I haven't set up the ci yet.

It should be something similar to 
https://github.com/Azure/azure-sdk-tools/blob/main/tools/tsp-client/ci.yml

Let me add the ci.yml first.